File: 7. Code Challenges/4. Count Vowels and Constants/Count_Vowels_Constants.js
let vowels          = "aeıioöuü".split("");
let constants       = "bcçdfgğhjklmnprsştvyzqwx".split("");
let countVowels     = 0;
let countConstants  = 0;

let str = prompt("String: ").toLowerCase().split("");
for (let i = 0; i < str.length; i++) {
    if (vowels.indexOf(str[i]) !== -1) {countVowels++;}
    if (constants.indexOf(str[i]) !== -1) {countConstants++;}
}
console.log(`Vowels   : ${countVowels}\nConstants: ${countConstants}`);

File: 7. Code Challenges/4. Count Vowels and Constants/Count_Vowels_Constants.py
string = input("String: ")
vowelsInStr = [i for i in string if i in "aeıioöuü"]
constantsInStr = [i for i in string if i in "bcçdfgğhjklmnprsştvyzqwx"]
print(f"Vowels:   {len(vowelsInStr)}")
print(f"Constants: {len(constantsInStr)}")